### PR TITLE
chore: add comments and syntax sugar for readability

### PR DESCRIPTION
**Pull request by thomas-le-goff**
Hi there!
I'd like to contribute to the project (if contributions are open). 
I started with small improvements (code style, how to, ...).

Depending on the needs and goals I can suggest more structural things.

I'd be glad to help on this project, which can be useful to many people!

Is it better to contribute via github or forge.aeif.fr?

### CODE REVIEW

**Comment by DegrangeM** [comment]
Hi (I will continue in english as you started, but as a french project we can also go back to french).

Thanks for your contribution. I was planning to clean up the code but I am happy you did it because it's not the funniest part to code :D

I looked quickly and it seems well written and not too far of what I was thinking. I still need to do a deep check to see if everything is okay before merging (this week or next week).

For contributing I prefer on github but it is also possible to contribute directly on the forge (but I will have to manually bring the patch on github as it's a one way sync Github > Forge.aeif.fr).

File: script.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+const SECOND = 60000; // milliseconds
```
Comment (DegrangeM) [inline script.js]: Shouldn't it be called "MINUTE" ?

Comment (thomas-le-goff) [reply script.js]: Of course, I think I was a bit too fast with the renaming

**Comment by DegrangeM** [comment]
The long press to switch to an other step was broken but I fixed it.

Thanks for the PR !

**Pull request merged**